test/parallel/test-readline-async-iterators.js

```diff
@@ -65,10 +65,13 @@ async function testMutual() {
       iterated = true;
 
       iteratedLines.push(k);
-      for await (const l of rli) {
-        iteratedLines.push(l);
+      try {
+        for await (const l of rli) {
+          iteratedLines.push(l);
+        }
+      } catch (err) {
+        assert.strictEqual(err.code, 'ERR_STREAM_ITERATOR_EXISTS');
```

Comment:
The comment above would need to be adjusted, moreover the `iterated` flag can be removed.